src/common.c

```diff
@@ -493,6 +500,38 @@ call_external(gchar** argv)
     return is_successful;
 }
 
+gboolean
+call_external_fork(gchar** argv)
+{
+    // Fork / exec external executable
+    pid_t pid = fork();
+    if (pid == 0) {
+        int x = execvp(argv[0], argv);
+        if (x == -1) {
+            auto_char char* cmd = g_strjoinv(" ", argv);
```

Comment:
```suggestion
            auto_gchar gchar* cmd = g_strjoinv(" ", argv);
```
Please use `auto_gchar` here, see #1819 for details.

Reply:
@sjaeckel Done.